sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -6,6 +6,8 @@
     <RootNamespace>SonarAnalyzer</RootNamespace>
     <CodeAnalysisRuleSet>..\SonarAnalyzer.ruleset</CodeAnalysisRuleSet>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <!-- This is a temporary workaround for https://github.com/Microsoft/vstest/issues/800 -->

```

Comment:
Looks like a good candidate for putting in a _Directory.Build.props_ file at the root of the solution.